ParkingApplication.api/Interfaces/IUserService.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ParkingApplication.Models;
+
+namespace ParkingApplication.Interfaces
+{
+    public interface IUserService
```

Comment:
Ce e cu inferata asta? Ai bagat-o in commit si nu e nimic in ea.

Reply:
Deleted, its done